File: mobile/src/main/java/com/chernowii/camcontrol/activity/SettingsActivity.java
package com.chernowii.camcontrol.activity;

import android.preference.PreferenceActivity;
import android.os.Bundle;

import com.chernowii.camcontrol.R;

public class SettingsActivity extends PreferenceActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.pref_general);
    }
}

File: mobile/src/main/java/com/chernowii/camcontrol/activity/SettingsActivity.kt
package com.chernowii.camcontrol.activity

import android.preference.PreferenceActivity
import android.os.Bundle

import com.chernowii.camcontrol.R

class SettingsActivity : PreferenceActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.pref_general)
    }
}


